.gitignore

```diff
@@ -2,6 +2,9 @@
 __pycache__/
 *.py[cod]
 
+# Editor tmp-files
+*.swp
+
```

Comment:
unnecessary change
-> this should be handled by your personal global `gitignore` - this is not project-specific .... 
see http://technicalpickles.com/posts/configure-git-to-globally-ignore-some-files/